api/Sources/Api/PairQL/Dashboard/Pairs/CombinedUsersActivitySummaries.swift

```diff
@@ -3,21 +3,14 @@ import PairQL
 
 struct CombinedUsersActivitySummaries: Pair {
   static let auth: ClientAuth = .admin
-  typealias Input = [DateRange]
```

Comment:
as noted in web pr, we don't take date ranges anymore, (we had hardcoded in the web repo to ask for most recent 14 days anyway) and now we want to get 14 days of normal activity, plus any flagged items